### PR TITLE
Bug fix business status

### PR DESCRIPTION
**Pull request by WordsPerMinute**
This fixed the bug, see https://developers.google.com/maps/deprecations#places_field_permanently_closed_-
Just required changing the variable name 'id' to 'place_id' in a few places when passing props/deconstructing objects.

### CODE REVIEW

File: src/components/SearchSidebar/PizzaPlaceCard.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const PizzaPlaceCard = (props) => {
         markerHoveredLocation,
         setSidebarHovered,
     } = props;
+
```
Comment (xbdell) [inline src/components/SearchSidebar/PizzaPlaceCard.js]: remove please

File: src/components/SearchSidebar/SearchSidebar.js [hunk 2]
```diff
@@ -20,13 +20,15 @@ const SearchSidebar = (props) => {
                 <div className={`PizzaLocationsContainer ${noPizzaLocations ? 'FlexCenter' : ''}`}>
                     {!noPizzaLocations &&
                         pizzaLocations.map((location) => (
-                            <div className="PizzaPlaceCardWrapper" key={location.id} id={`shop-marker-${location.id}`}>
+                            <div className="PizzaPlaceCardWrapper" key={location.place_id} id={`shop-marker-${location.place_id}`}>
+                                {/* {console.log(location)} */}
```
Comment (xbdell) [inline src/components/SearchSidebar/SearchSidebar.js]: remove please

File: src/components/PizzaMap/ShopMarker.js [hunk 3]
```diff
@@ -4,7 +4,8 @@ import './ShopMarker.scss';
 const ShopMarker = (props) => {
     const { setSelectedLocation, location, setMarkerHoveredLocation, sidebarHovered } = props;
 
-    const { id } = location;
+    const { place_id } = location;
```
Comment (xbdell) [inline src/components/PizzaMap/ShopMarker.js]: `const { place_id: id } = location;`

File: src/components/SearchSidebar/PizzaPlaceCard.js (hunk 1 above)
Comment (WordsPerMinute) [reply src/components/SearchSidebar/PizzaPlaceCard.js]: addressed, updated commit

File: src/components/SearchSidebar/SearchSidebar.js (hunk 2 above)
Comment (WordsPerMinute) [reply src/components/SearchSidebar/SearchSidebar.js]: addressed, updated commit

File: src/components/PizzaMap/ShopMarker.js (hunk 3 above)
Comment (WordsPerMinute) [reply src/components/PizzaMap/ShopMarker.js]: addressed, updated commit

**Pull request merged**